kb/disorders/Frontotemporal_Dementia.yaml

```diff
@@ -0,0 +1,184 @@
+    and causes neuronal degeneration primarily in frontal and temporal regions.
+  cell_types:
+  - preferred_term: neuron
+    term:
+      id: CL:0000540
+      label: neuron
+  downstream:
+  - target: Neuronal Death
+  - target: Behavioral and Language Changes
+- name: Frontal and Temporal Lobe Atrophy
+  description: >
+    Progressive neuronal loss and atrophy predominantly affecting the frontal and
+    temporal lobes, with patterns varying by genetic subtype. MAPT mutations show
+    temporal predominance, while GRN and C9orf72 mutations may show more variable
+    patterns. This structural degeneration underlies the behavioral and language deficits.
+  cell_types:
+  - preferred_term: neuron
+    term:
+      id: CL:0000540
+      label: neuron
+  downstream:
+  - target: Behavioral Variant FTD
+  - target: Primary Progressive Aphasia
+  evidence:
+  - reference: PMID:26473392
+    supports: SUPPORT
+    snippet: MAPT patients were younger than other groups, and showed more frequent behavioural disinhibition, repetitive and stereotyped behaviours, semantic impairment and temporal predominance of atrophy.
+    explanation: This demonstrates the genotype-specific patterns of brain atrophy and their relationship to clinical phenotypes in frontotemporal dementia.
+- name: Neuronal Death
+  description: >
+    Progressive loss of neurons in frontal and temporal cortices due to protein
+    aggregation toxicity, leading to brain atrophy and functional decline. The
+    neurodegeneration progresses over years to decades, with biomarker changes
+    detectable up to 30 years before symptom onset in genetic cases.
+  cell_types:
+  - preferred_term: neuron
+    term:
+      id: CL:0000540
+      label: neuron
+  evidence:
+  - reference: PMID:30711676
+    supports: SUPPORT
+    snippet: We observed that biological changes and intracortical facilitation transmission abnormalities significantly antecede the emergence of clinical symptoms of at least 3 decades.
+    explanation: This shows that neurobiological changes underlying neuronal dysfunction begin decades before clinical symptoms in genetic FTD.
+phenotypes:
+- category: Neurological
+  name: Behavioral Changes
+  frequency: VERY_FREQUENT
+  diagnostic: true
+  notes: Prominent behavioral changes including disinhibition, apathy, loss of empathy, compulsive behaviors, and dietary changes. Particularly characteristic of behavioral variant FTD.
+  evidence:
+  - reference: PMID:26473392
+    supports: SUPPORT
+    snippet: MAPT patients were younger than other groups, and showed more frequent behavioural disinhibition, repetitive and stereotyped behaviours, semantic impairment and temporal predominance of atrophy.
+    explanation: This confirms that behavioral disinhibition and repetitive stereotyped behaviors are core clinical features of genetic FTD, particularly in MAPT mutations.
+- category: Neurological
```

Comment:
This is near perfect. But ideally we would capture the "non-fluent" part in the title/description even if the ontology term is more general. See things-v-strings discussion. We may want to use this example as we circle back on this